php/php-pcov/Portfile

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               github 1.0
+PortGroup               php 1.1
+
+name                    php-pcov
+version                 1.0.11
+categories-append       devel
+platforms               darwin
+maintainers             {ryandesign @ryandesign} openmaintainer
```

Comment:
did @ryandesign agree to maintain this port? If not, you can add yourself as the maintainer or leave it as `nomaintainer`

Reply:
Nope, @ryandesign didn't agree. I just added him coz in other ports he has ended becoming the maintainer. I've switched it to nomaintainer in fixup commit. Thanks!

Reply:
While usually people should opt in to maintaining a port, I am happy to be made the maintainer of straightforward php modules like this.

Reply:
Thanks @ryandesign , have added you back!